tests/unit/classifier/GaussianProcessClassification_unittest.cc

```diff
@@ -3525,19 +3523,17 @@ TEST(GaussianProcessClassificationUsingSingleFITCLaplacian,get_variance_vector)
 	CBinaryLabels* labels_train=new CBinaryLabels(lab_train);
 
 	// choose Gaussian kernel with sigma = 2 and zero mean function
-	float64_t ell=1.0;
-
-	CLinearARDKernel* kernel=new CGaussianARDFITCKernel(10, 2*ell*ell);
+	CGaussianARDFITCKernel* kernel=new CGaussianARDFITCKernel(10);
```

Comment:
I think it would be better to be explict about parameters here rather than using default values?

Also, can we get rid of this annoying kernel cache size parameter? This should be set to default instead